### DISCUSSION TITLE
UI y UX

### DISCUSSION THREAD

**Discussion by AndresAVargas** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#113] [Q&A]
Compañeros buen día, espero me puedan ayudar como UX elabore esto
![flujo](https://user-images.githubusercontent.com/88987587/155617997-812133bf-b7aa-4f86-bc30-dbddce2f338d.png)
y la verdad no se si lo que elabore esta bien o era otra cosa, además no entiendo que tengo que hacer para el UI, no se si sean tan amables de explicarme el UI

**Comment by Sergioavr** [comment]
A lo que entendí de los lives y de lo que he leído, la UI se orienta mas que nada al diseño y todo lo que tiene que ver con lo que el usuario ve, en este caso, en el UI podrías elaborar un boceto de cómo se vería la interfaz de la app en algún programa de los que puso el mc en los links de las mismas practicas o utilizar alguno que tu conozcas o se te haga más fácil, mientras que UX es la manera en que el usuario va a interactuar, en la cual debemos de promover que sea de lo mas didáctico e intuitivo posible para que el usuario tenga una interacción sencilla con la app, y me parece que si esta bien lo que hiciste, sin embargo, aunque realmente el mc comentó que es más que nada para que practiquemos, podrías expandir más el mapa y crear un flujo más amplio y detallado, aunque claro, no sé si estoy en lo correcto, saludos. 👍
